WS2012R2/lisa/setupscripts/INST_LIS_TimeSync.ps1

```diff
@@ -160,13 +160,13 @@ function GetUnixVMTime([String] $sshKey, [String] $ipv4)
 
     #
     # now=`date "+%m/%d/%Y %H:%M:%S%p"
-    # returns 04/27/2012 16:10:30PM
+    # returns 04/27/2012/16:10:30
```

Comment:
The previous comment was correct, on a RHEL 7.x the returned format from "+%m/%d/%Y %H:%M:%S%p" is "08/01/2016 04:13:04AM", so the PM/AM format appears.